pkg/regionspan/region_range_lock.go

```diff
@@ -298,7 +299,11 @@ func (l *RegionRangeLock) LockRange(startKey, endKey []byte, regionID, version u
 		var res1 LockRangeResult
 		for {
 			for _, ch := range signalChs1 {
```

Comment:
Could reading from signalChs1 block when the kvclient is being cancelled?

Reply:
Do you mean range for `signalChs1`, it is a chan slice